dotCMS/src/main/java/com/dotmarketing/portlets/workflows/actionlet/SaveContentAsDraftActionlet.java

```diff
@@ -29,16 +31,27 @@
 public class SaveContentAsDraftActionlet extends WorkFlowActionlet {
```

Comment:
I think SaveContentlet needs the same

Reply:
done https://github.com/dotCMS/core/pull/18300/commits/1aa3444ac01b694697c3580a424d7e3566aabc35#diff-7834fdbf27d8f42237ae8fb6b94bee62R71